app/state/middlewares/navigation/navigation.js

```diff
@@ -15,17 +15,16 @@ const middleware = navigationActions => store => next => action => {
     return next(action)
   }
 
-  const {
-    routeName: currentRouteName
-  } = navigator.state.nav.routes[navigator.state.nav.index]
+  const route = navigator.state.nav.routes[navigator.state.nav.index]
+  const fullRouteName = `${route.routeName}.${route.routes[route.index].routeName}`
```

Comment:
Is `${route.routeName}.${route.routes[route.index].routeName}` a hack or a standard way of doing it?

Reply:
Not a standard. So far it works but probably we can use a better format like `/a/b/c`

Reply:
Use slash `/` as routeNames delimiter a5971f9